config/webhook/manifests.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: validating-webhook-configuration
+webhooks:
+  - clientConfig:
+      caBundle: Cg==
+      service:
+        name: webhook-service
+        namespace: system
+        path: /validate-metal3-io-v1alpha1-baremetalhost
+    failurePolicy: Ignore
```

Comment:
I guess we have got this far without validation at all, but to my mind this ought to be something we can rely on in future, not something that will _maybe_ enforce some invariants. Are there other opinions on this?